test/runtests.jl

```diff
@@ -400,6 +402,8 @@ include("setup_tests.jl")
             @test cleanup_named_tuple_type(string(typeof((;)))) == "NamedTuple{(),Tuple{}}"
             @test cleanup_named_tuple_type("@NamedTuple{x::Int, y::Int}") ==
                   "NamedTuple{(:x,:y),Tuple{Int,Int}}"
+            @test cleanup_named_tuple_type("@NamedTuple{x::Int}") ==
```

Comment:
we should also probably test a nested named tuple here where at least one leaf is a singleton NamedTuple